docs/GettingStarted.md

```diff
@@ -0,0 +1,14 @@
+# Getting Started
```

Comment:
Should we even have this file?

Reply:
The main thing I'm angling for on the README is that when a user views the package on nuget.org, there's _something_ there they can copy/paste to get started. Some recent experiences I had with python/rust devs have really impressed on me the need to have something actionable. 

The other intended use for this file is a link from a yet-to-be-written blog post, so I still want something here for users to land on.

I guess what I'm saying is I'm ok with a little bit of duplication, given the distinct audiences I'm angling each file for.

Reply:
I think this is ok, and if they wind up staying identical, git supports symlinks . . .